CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## UNRELEASED
+
+* Adds option to search component to allow custom name attributes on the input field
```

Comment:
Can you update this to match the PR name and include the PR number please? i.e. `Allow custom name attribute on search input (PR #787)`